observability/readme.md

```diff
@@ -1,132 +1,33 @@
 
-Read more [here](pitfalls.md) to understand what to watch out for while designing and building an observable system.
+- [Tooling](tools/readme.md)
+- [Observability As Code](observability-as-code.md)
+- [Recommended Practices](best-practices.md)
 
 ## Recipes
 
-### Application Insights/ASP.NET
-
-[Github Repo](https://github.com/Azure-Samples/application-insights-aspnet-sample-opentelemetry), [Article](https://devblogs.microsoft.com/aspnet/observability-asp-net-core-apps/).
-
-### Example: Setup Azure Monitor dashboards and alerts with Terraform
-
-[Github Repo](https://github.com/buzzfrog/azure-alert-dashboard-terraform)
-
-### On-premises Application Insights
-
-[On-premise Application Insights](https://github.com/c-w/appinsights-on-premises) is a service that is compatible with Azure App Insights, but stores the data in an in-house database like PostgreSQL or object storage like [Azurite](https://github.com/Azure/Azurite).
-
-On-premises Application Insights is useful as a drop-in replacement for Azure Application Insights in scenarios where a solution must be cloud deployable but must also support on-premises disconnected deployment scenarios.
-
-On-premises Application Insights is also useful for testing telemetry integration. Issues related to telemetry can be hard to catch since often these integrations are excluded from unit-test or integration test flows due to it being non-trivial to use a live Azure Application Insights resource for testing, e.g. managing the lifetime of the resource, having to ignore old telemetry for assertions, if a new resource is used it can take a while for the telemetry to show up, etc. The On-premise Application Insights service can be used to make it easier to integrate with an Azure Application Insights compatible API endpoint during local development or continuous integration without having to spin up a resource in Azure. Additionally, the service simplifies integration testing of asynchronous workflows such as web workers since integration tests can now be written to assert against telemetry logged to the service, e.g. assert that no exceptions were logged, assert that some number of events of a specific type were logged within a certain time-frame, etc.
-
-### Azure DevOps Pipelines Reporting with Power BI
-
-The [Azure DevOps Pipelines Report](https://github.com/Azure-Samples/powerbi-pipeline-report) contains a [Power BI](https://docs.microsoft.com/en-us/power-bi/fundamentals/power-bi-overview) template for monitoring project, pipeline, and pipeline run data from an Azure DevOps (AzDO) organization.
-
-This dashboard recipe provides observability for AzDO pipelines by displaying various metrics (i.e. average runtime, run outcome statistics, etc.) in a table. Additionally, the second page of the template visualizes pipeline success and failure trends using Power BI charts. Documentation and setup information can be found in the project README.
+- [Observability in Machine Learning](ml-observability.md)
```

Comment:
ML and Delivery pipelines don't really fall under the "Recipes" section. What do you think about moving these two to "Facets of Observability" - in reality, these are explaining about how to observe in a particular component, say delivery pipelines or in ML. I think moving these into Recipes gives an impression that these are 'show and tells' of how to do observability. Thoughts?

Reply:
I like the term "Facets" +1 on that.

Reply:
I like the term "Fasets" too! Updated  and left recipes under this category